.github/workflows/node.yaml

```diff
@@ -175,17 +174,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: Get the Docker tag
-        id: docker-tag
-        uses: yuya-takeyama/docker-tag-from-github-ref-action@750c706417ebb9a701022906a9725d21332508f7
-        with:
-          remove-version-tag-prefix: false
       - name: Determine images to publish
         id: image-tags
         # TODO - image needs changing...
         run: |
           IMAGES=
-          DOCKER_TAG=${{ steps.docker-tag.outputs.tag }}
+          if [[ ${{ github.ref_name }} == "master" || ${{ github.ref_name }} == "main" ]]
+            DOCKER_TAG=latest
+          else
+            DOCKER_TAG=${{ github.ref_name }}
```

Comment:
Are special characters allowed in the docker tag or should it have the same filter applied as elsewhere?

Reply:
It appears they are not, but it didn't have it before so I didn't add it, probably should!
The docker website says:
`A tag name must be valid ASCII and may contain lowercase and uppercase letters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.`

I wonder if we should add a check for length as well then...